components/mailbox/src/Mailbox.svelte

```diff
@@ -499,6 +503,21 @@
     loading = false;
   }
   //#endregion pagination
+
+  function dispatchDraft(event) {
+    const { thread } = event.detail;
+    const message = thread.drafts[0];
+    const value = {
+      to: message.to,
+      cc: message.cc,
+      bcc: message.bcc,
+      from: message.from,
+      subject: message.subject,
+      body: message.body,
+    };
+
+    dispatchEvent("draftThreadEvent", { event, message, thread, value });
```

Comment:
copying values for dispatched reply and reply all events, so they can share a function in mailbox's index.html